test/widgets/message_list_test.dart

```diff
@@ -69,8 +69,8 @@ void main() {
 
     testWidgets('basic', (tester) async {
       await setupMessageListPage(tester, foundOldest: false,
-        messages: List.generate(100, (i) => eg.streamMessage(id: 950 + i, sender: eg.selfUser)));
-      check(itemCount(tester)).equals(101);
+        messages: List.generate(200, (i) => eg.streamMessage(id: 950 + i, sender: eg.selfUser)));
+      check(itemCount(tester)).equals(201);
```

Comment:
Huh, interesting: I guess when there are a lot of consecutive short messages with the same sender and recipient, we now save a lot of vertical space, so this test needs more messages to cover as much vertical space as it wants to.

Reply:
Yep, exactly. Without this change, these messages take up so little vertical space that the message list promptly decides it needs to fetch more older messages, disrupting the sequence this test was going for.